### PR TITLE
Fixed eslint issues ocomponents [A* - M*]

### PR DESCRIPTION
**Pull request by tw-sumitsj**

### CODE REVIEW

File: src/components/DiagnosticReport/DiagnosticReportComponent.js [hunk 1]
```diff
@@ -42,38 +42,33 @@ const DiagnosticReportComponent = ({ data, consentReqId }) => {
...
     const referenceList = [];
-    results
-      ? results.map((result) => {
-        const resource = result.targetResource;
-        referenceList.push(resource);
-      })
-      : undefined;
-    return referenceList.filter((ref) => (ref.resourceType = resourceType));
+    if (results) results.forEach((result) => referenceList.push(result.targetResource));
+    return referenceList.filter((ref) => (ref.resourceType === resourceType));
```
Comment (dheeraj-p) [inline src/components/DiagnosticReport/DiagnosticReportComponent.js:46]: Looks like there is some change in logic here.
It was `(ref) => (ref.resourceType = resourceType)`, which looks wrong. I am wondering how it was working before.

File: src/components/Medication/MedicationRequestsComponent.js [hunk 2]
```diff
@@ -247,6 +27,7 @@ const findMedicationName = function (medication) {
   return 'Unspecified';
 };
 
+// eslint-disable-next-line max-len
```
[the remark this answers is not in the document]
Comment (dheeraj-p) [reply src/components/Medication/MedicationRequestsComponent.js:30]: We can decide max line length and use that.

[the remark this answers is not in the document]
Comment (tw-sumitsj) [reply src/components/Medication/MedicationRequestsComponent.js:30]: I tried to break this line into two, but then I was getting error for another rule. So, decided to disable it for that specific line.

File: src/components/DiagnosticReport/DiagnosticReportComponent.js (hunk 1 above)
Comment (tw-sumitsj) [reply src/components/DiagnosticReport/DiagnosticReportComponent.js:46]: I don't think filtering was working before. Can you please check on dev?

Comment (dheeraj-p) [reply src/components/DiagnosticReport/DiagnosticReportComponent.js:46]: This looks correct.

**Pull request merged**